checkstyle-script.bat

```diff
@@ -0,0 +1 @@
+gradlew checkstyleMain checkstyleTest
```

Comment:
Are you sure you want to include this?

Reply:
oops, that's the old trivial script that i wrote. I forgot that my git ignore was the old one when i checked out this new branch. Will add bat files to the gitignore instead.